components/latex_block/latex_block.test.tsx

```diff
@@ -12,28 +12,34 @@ describe('components/LatexBlock', () => {
         enableLatex: true,
     };
 
-    test('should match snapshot', () => {
+    test('should match snapshot', async () => {
         const wrapper = shallow(<LatexBlock {...defaultProps}/>);
+        await import('katex'); //manually import katex
+        wrapper.update();
```

Comment:
Is this needed? I'd expect the snapshot to be updated automatically once the import statement causes the `setState`